Generals/Code/Libraries/Source/WWVegas/WWDebug/wwmemlog.cpp

```diff
@@ -214,33 +216,41 @@ static bool							_MemLogAllocated = false;
 */
 void * Get_Mem_Log_Mutex(void)
 {
+#ifdef _WIN32
 	if (_MemLogMutex == NULL) {
 		_MemLogMutex=CreateMutex(NULL,false,NULL);
 		WWASSERT(_MemLogMutex);
 	}
 	return _MemLogMutex;
+#else
+	return NULL;
+#endif
 }
 
 void Lock_Mem_Log_Mutex(void)
 {
+#ifdef _WIN32
 	void * mutex = Get_Mem_Log_Mutex();
 #ifdef DEBUG_CRASHING
 	int res =
 #endif
 		WaitForSingleObject(mutex,INFINITE);
 	WWASSERT(res==WAIT_OBJECT_0);
 	_MemLogLockCounter++;
+#endif
 }
```

Comment:
I think some alternative implementation should be provided or some kind runtime error should be produced...

Reply:
Why not just replace this version of wwmemlog with the version from ZH which seemingly is more linux friendly?